.github/twitter.py

```diff
@@ -0,0 +1,50 @@
+import tweepy
+import requests
+import xml.etree.ElementTree as ET
+import os
+import sys
+
+def main():
+    client = tweepy.Client(
+        consumer_key=os.getenv("API_KEY"),
```

Comment:
better to use os.environ since that will throw an error if the key doesn't exist. This is ideal here since the key is essential to the function